src/components/OptionsSelector.js

```diff
@@ -184,6 +184,7 @@ class OptionsSelector extends Component {
                 </View>
                 <OptionsList
                     ref={el => this.list = el}
+                    optionHoveredStyle={[styles.modalLinkHover]}
```

Comment:
Please refactor `styles.modalLinkHovered` and it's usages (including this one) to something more generic like `styles.hoveredComponentBG`.

Reply:
Also, there's no need for square brackets here, you could just do:

```
optionHoveredStyle={styles.hoveredComponentBG}
```
```